lte/gateway/python/setup.py

```diff
@@ -137,9 +137,9 @@
             # Keep grpcio and grpcio-tools on same version for now
             # If you update this version here, you probably also want to
```

Comment:
This seems outdated - I do not see grpc setup in lte/gateway/python/Makefile

Reply:
I removed the comment